osu.Game.Rulesets.Catch/Mods/CatchModHidden.cs

```diff
@@ -54,7 +54,7 @@ private void fadeOutHitObject(DrawableCatchHitObject drawable)
             var offset = hitObject.TimePreempt * fade_out_offset_multiplier;
             var duration = offset - hitObject.TimePreempt * fade_out_duration_multiplier;
 
-            using (drawable.BeginAbsoluteSequence(hitObject.StartTime - offset, true))
+            using (drawable.BeginAbsoluteSequence(hitObject.StartTime - offset))
```

Comment:
Not sure why stop at removing `Begin{Absolute,Delayed}Sequence(..., true)` calls only when it's hidden - should really be a separate cleanup PR, removing these across the entire repository instead of by portion, but I guess it makes *some* sense scope-wise in the context of this PR.